src/server.ts

```diff
@@ -1,9 +1,19 @@
+import { AddressInfo } from 'net';
+
 import App from './app';
 
 (() => {
   App.startServer().then(
-    () => { },
-    // eslint-disable-next-line no-console
-    (e) => { console.error(e); },
+    (server) => {
+      // eslint-disable-next-line no-console
+      console.log(
+        'Server is listening on port: ',
+        (server.address() as AddressInfo).port,
+      );
```

Comment:
We should not be using `console.log` at any place. We have a common module in place which takes care of logging.

Reply:
Removed this